From 1dd980777bd32853437095d78c650318078e5ac2 Mon Sep 17 00:00:00 2001
From: Lawrence Rust <lvr@softsystem.co.uk>
Date: Tue, 13 Mar 2012 17:47:34 +0100
Subject: [PATCH 004/227] Profile build on Windows and MacOSX don't require Qt
 debug libs

At present profile builds pull in debug builds of Qt.  This
causes link errors on Windows and MacOSX unless the debug libs
are available.  This patch removes debug from CONFIG on these
platforms.

Signed-off-by: Lawrence Rust <lvr@softsystem.co.uk>
---
 mythplugins/settings.pro |    2 +-
 mythtv/settings.pro      |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/mythplugins/settings.pro b/mythplugins/settings.pro
index d3c944f..fdaeefa 100644
--- a/mythplugins/settings.pro
+++ b/mythplugins/settings.pro
@@ -30,7 +30,7 @@ QMAKE_CXXFLAGS += $$CONFIG_DIRECTFB_CXXFLAGS
 QMAKE_CXXFLAGS_SHLIB = -DPIC -fPIC
 QMAKE_CXXFLAGS += $$ECXXFLAGS
 
-profile:CONFIG += debug
+profile:!win32:!macx:CONFIG += debug
 
 QMAKE_CXXFLAGS_RELEASE = $$OPTFLAGS -fomit-frame-pointer
 release:contains( TARGET_ARCH_POWERPC, yes ) {
diff --git a/mythtv/settings.pro b/mythtv/settings.pro
index 407f58a..3263ad9 100644
--- a/mythtv/settings.pro
+++ b/mythtv/settings.pro
@@ -113,7 +113,7 @@ macx: QMAKE_CFLAGS_STATIC_LIB += -fno-common
 QMAKE_CFLAGS   += $$CPPFLAGS   $$CFLAGS
 QMAKE_CXXFLAGS += $$CXXPPFLAGS $$ECXXFLAGS
 
-profile:CONFIG += debug
+profile:!win32:!macx:CONFIG += debug
 
 # Allow compilation with Qt Embedded, if Qt is compiled without "-fno-rtti"
 QMAKE_CXXFLAGS -= -fno-exceptions -fno-rtti
-- 
1.7.9.5

